amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
```

Comment:
Ugh - which is the current schema in which this table leave?
In the current layout there are 2 schemas: ag and barcodes. If the tables are in the barcodes schema it is ok to leave it there, but if the tables are in the ag schema that need to be changed (either to a new schema or to the barcode schema).

Sorry about this I didn't think about it earlier.